downstream/assemblies/platform/assembly-controller-glossary.adoc

```diff
@@ -231,7 +231,7 @@ The calendar of dates and times for which a job should run automatically.
 See *Distributed Job*.
 
 [discrete]
-=== Source Credential::
+*Source Credential*::
```

Comment:
I think the reason the formatting is broken is because of the two `::`. I wonder if it would make sense to match the formatting of the other sections and have it be `=== Source Credential` instead?

Reply:
Oh I see, in the ticket it asked for it to be in bold, or that was my understanding of it, I will attach it below: https://issues.redhat.com/browse/AAP-24465

Reply:
Ah, it might be worth checking with Donna to see if that is the intended result?

Reply:
Okay perfect, thank you :)

Reply:
Hey @KylaF8, @michellemacrh is correct that you just need to remove the 2 colons at the end. This should be in the file as follows:

=== Source Credential

Reply:
Hi Donna, okay perfect thank you. So instead of having it as: *Source Credential*:: 
Have it as: ===Source Credential

Reply:
@KylaF8 Yes, but don't forget the space between === and Source Credential. So, 

=== Source Credential